dist/ng-token-auth.js

```diff
@@ -583,10 +583,16 @@ angular.module('ng-token-auth', ['ipCookie']).provider('$auth', function() {
               return name || this.getSavedConfig();
             },
             getSavedConfig: function() {
-              var c, key;
+              var c, error, hasLocalStorage, key;
               c = void 0;
               key = 'currentConfigName';
-              if ($window.localStorage) {
+              hasLocalStorage = false;
+              try {
+                hasLocalStorage = !!$window.localStorage;
+              } catch (_error) {
+                error = _error;
```

Comment:
1. Which unused variable do you mean?  If you mean the 'error', it seemed like coffeescript required that.  I'm no good at coffeescript, so I my have done it wrong.
1. I def don't want to throw an error, since the whole point of this PR is to suppress the error.
1. I don't think a console.warn is necessary.  This library is supposed to support situations with no local storage, so it does not really require a warning, IMO.